react/PWAContext.js

```diff
@@ -54,14 +69,17 @@ const PWAProvider = ({ rootPath, children, data = {} }) => {
   const context = useMemo(() => {
     if (pwaSettings) {
       const { disablePrompt, promptOnCustomEvent } = pwaSettings
+      const isIOS = navigator && !!navigator.userAgent.match(/(iPod|iPhone|iPad)/)
+
       return {
         showInstallPrompt,
         settings: {
-          promptOnCustomEvent: disablePrompt ? '' : promptOnCustomEvent
+          promptOnCustomEvent: (disablePrompt || isIOS || installDismissed || alreadyInstalled) ? '' 
+            : promptOnCustomEvent
         }
       }
     }
-  }, [showInstallPrompt, pwaSettings])
+  }, [showInstallPrompt, pwaSettings, alreadyInstalled])
```

Comment:
are we missing the `installDismissed` variable here?

Reply:
I forgot, thanks!